README.rst

```diff
@@ -9,6 +9,11 @@ Use the API by instantiating it, and then calling the method through dotted nota
 	$disqus = new DisqusAPI($secret_key);
 	$disqus->trends->listThreads();
 
+Advanced Example	
+~~~~~~~~~
+* How to integrate Disqus API with your CMS 
+	(http://codexc.com/blog/2015/03/disqus-comments-moderation-in-your-cms/)
```

Comment:
Would prefer to move this into a `docs` directory, or the GitHub Wiki. If your website goes down, the documentation will be lost.